website/index.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Faster+One|Ranchers&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css"/>
+    <noscript>You need to enable JavaScript to use the full functionality of the site.</noscript>
+    <script src="main.js"></script>
+    <title>Crow Fact Oracle&trade;</title>
+  </head>
+  <body>
+    <div class="title"><div class="title-photo"></div></div>
+    <div class="container-fluid menu">
+        <div class="row">
+            <div class="col-sm-4 menu-item" onclick="renderPage('getCrows', 0)">Check Out These Crows</div>
+            <div class="col-sm-4 menu-item" onclick="renderPage('getFacts', 1)">Learn Stuff About Them</div>
+            <div class="col-sm-4 menu-item" onclick="renderPage('teachUs', 2)">Share Your Cool Crow Fact</div>
+        </div>
+    </div>
+
+    <div id="getCrows" class="content hidden">
+    </div>
+
+    <div id="getFacts" class="content hidden">
+    </div>
+
+    <div id="teachUs" class="content hidden">
+    </div>
```

Comment:
Last side note: it's not really best practice to hide DOM elements. If you are wanting to hide them, they should just be removed from the DOM completely. For this use case, we could just use a single DOM element and mutate that whenever the end user click on one of the three menu item elements

Reply:
@bpas247 can you link to code examples or tutorials for this? This is bleeding into the realm of stuff that I don't know how to do, so some examples would be helpful.

Reply:
Eh, on second thought this is probably fine. Removing DOM nodes would mess up the layout, and make styling more difficult